### DISCUSSION TITLE
y2mate.is: detection

### DISCUSSION THREAD

**Discussion by igtbap** [uBlockOrigin/uAssets#25304] [General]
It forces me to disable adblocker but I had to add `y2mate.is##+js(set, MutationObserver.prototype.observe, noopFunc)` to my filter and it worked fine (Finally, adblocker detection stopped working), I think their filters is outdated.
Can we update their filter?

**Comment by gorhill** [comment]
I navigated to `https://en.y2mate.is/v94/` and I didn't experience detection. How is the detection triggered?

**Reply by igtbap** [reply]
I fetched it with YouTube video URL and it forces me to disable adblock before I can download it.

**Reply by gorhill** [reply]
I pasted a Youtube URL, clicked _Start_ button, then clicked _Convert_ button, then clicked _Download_ button and no detection was triggered.

When you file a report for a specific site, use the _chat_ icon in the popup panel so that we know more about your configuration, and you know more about what else external to uBO could be the issue as you are asked to ensure your are using supported configuration.

**Reply by igtbap** [reply]
If you don't use my custom filter, the detection will trigger and all _Convert_ buttons will disable (I use Microsoft Edge)

**Reply by gorhill** [reply]
Submit a report through the _chat_ icon while being on the site, after the detection occurs. Without that step, I can't help you further.

**Reply by igtbap** [reply]
I found it. Thanks!
